exporter/exporter.go

```diff
@@ -374,7 +374,8 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
 		"master_link_up":                               {txt: "Master link status on Redis slave", lbls: []string{"master_host", "master_port"}},
 		"master_sync_in_progress":                      {txt: "Master sync in progress", lbls: []string{"master_host", "master_port"}},
 		"number_of_distinct_key_groups":                {txt: `Number of distinct key groups`, lbls: []string{"db"}},
-		"script_values":                                {txt: "Values returned by the collect script", lbls: []string{"key"}},
+		"script_result":                                {txt: "Result of the collect script evaluation", lbls: []string{"filename"}},
```

Comment:
As we discussed, I added new metric `script_result`, which will indicate if evaluation of Lua failed. 
So far I set 0 - fail, 1 - ok, 2 - empty results.